cmd/utils/flags.go

```diff
@@ -291,12 +293,24 @@ var (
 		Name:  "statedb.use-txpool-cache",
 		Usage: "Enables caching of nonce and balance for txpool.",
 	}
-	TrieCacheLimitFlag = cli.IntFlag{
+	TrieNodeCacheTypeFlag = cli.StringFlag{
+		Name:  "statedb.cache.type",
```

Comment:
How about starting with `cache.statedb` or `state.cache`?
When it starts with `statedb.cache`, it looks like it has connections to `statedb.use-cache`. `cache.statedb` looks like it matched naming with db flag and `state.cache` with `state.trie-cache-limit`.

Reply:
We need a discussion about it. I think `statedb` is the correct one because the cache is a cache of stateDB and it is defined in `statedb` package. 
I know the name is not match up with `state.trie-cache-limit`. I thought we need to change `state.trie-cache-limit` to `statedb.trie-cache-limit` or something else, but I didn't change it because of backward compatibility. 
@winnie-byun Which one do you think correct if you don't care about the consistency with previous flags.

Reply:
Actually, I agree with you. Changing `state.trie-cache-limit` to `statedb.trie-cache-limit` seems right. However, if we are not planning to change `state.trie-cache-limit`, it would be better to make consistency with previous settings.

Reply:
@winnie-byun Let's discuss the change. I think we can add `statedb.cache.fast.size` flag doing the same thing with `state.trie-cache-limit`. Then, we can have consistency changing the help description to show only the new flag. @KimKyungup @ehnuje @jeongkyun-oh @yoomee1313 How do think about the change?

Reply:
Or, how about commenting `deprecated` for `state.trie-cache-limit`?

Reply:
I think the prefix `statedb.cache.` is ok.
Now we applied auto scale fast cache.
`state.trie-cache-limit` flag is used rarely.
So I think we can change the name.
If someone runs the new binary with `state.trie-cache-limit`, it will not run.

Reply:
Okay. I will handle this issue in other PR. Please, give more opinions until this PR closed.